[2025-06-06T03:16:56.758+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-06T03:16:56.778+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pokemon_partitioned_to_kafka.fetch_pokemon_and_send_partitioned_to_kafka scheduled__2025-06-06T03:02:00+00:00 [queued]>
[2025-06-06T03:16:56.784+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pokemon_partitioned_to_kafka.fetch_pokemon_and_send_partitioned_to_kafka scheduled__2025-06-06T03:02:00+00:00 [queued]>
[2025-06-06T03:16:56.784+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-06-06T03:16:56.792+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_pokemon_and_send_partitioned_to_kafka> on 2025-06-06 03:02:00+00:00
[2025-06-06T03:16:56.796+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1568) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-06T03:16:56.796+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pokemon_partitioned_to_kafka', 'fetch_pokemon_and_send_partitioned_to_kafka', 'scheduled__2025-06-06T03:02:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/pkapi_to_kafka_dag.py', '--cfg-path', '/tmp/tmpsny5gioz']
[2025-06-06T03:16:56.797+0000] {standard_task_runner.py:63} INFO - Started process 1570 to run task
[2025-06-06T03:16:56.798+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask fetch_pokemon_and_send_partitioned_to_kafka
[2025-06-06T03:16:56.806+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-06T03:16:56.826+0000] {task_command.py:426} INFO - Running <TaskInstance: pokemon_partitioned_to_kafka.fetch_pokemon_and_send_partitioned_to_kafka scheduled__2025-06-06T03:02:00+00:00 [running]> on host 856ef77f0083
[2025-06-06T03:16:56.872+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pokemon_partitioned_to_kafka' AIRFLOW_CTX_TASK_ID='fetch_pokemon_and_send_partitioned_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T03:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-06T03:02:00+00:00'
[2025-06-06T03:16:56.873+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-06T03:17:02.181+0000] {logging_mixin.py:188} INFO - Error fetching Pokémon ID 1: HTTPSConnectionPool(host='pokeapi.co', port=443): Read timed out. (read timeout=5)
[2025-06-06T03:17:02.895+0000] {logging_mixin.py:188} INFO - Produced: ivysaur with key=grass
[2025-06-06T03:17:03.375+0000] {logging_mixin.py:188} INFO - Produced: venusaur with key=grass
[2025-06-06T03:17:03.832+0000] {logging_mixin.py:188} INFO - Produced: charmander with key=fire
[2025-06-06T03:17:09.289+0000] {logging_mixin.py:188} INFO - Error fetching Pokémon ID 5: HTTPSConnectionPool(host='pokeapi.co', port=443): Read timed out.
[2025-06-06T03:17:09.524+0000] {logging_mixin.py:188} INFO - Produced: charizard with key=fire
[2025-06-06T03:17:09.970+0000] {logging_mixin.py:188} INFO - Produced: squirtle with key=water
[2025-06-06T03:17:10.404+0000] {logging_mixin.py:188} INFO - Produced: wartortle with key=water
[2025-06-06T03:17:10.837+0000] {logging_mixin.py:188} INFO - Produced: blastoise with key=water
[2025-06-06T03:17:11.282+0000] {logging_mixin.py:188} INFO - Produced: caterpie with key=bug
[2025-06-06T03:17:11.758+0000] {logging_mixin.py:188} INFO - Produced: metapod with key=bug
[2025-06-06T03:17:12.677+0000] {logging_mixin.py:188} INFO - Error fetching Pokémon ID 12: HTTPSConnectionPool(host='pokeapi.co', port=443): Max retries exceeded with url: /api/v2/pokemon/12/ (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')))
[2025-06-06T03:17:12.930+0000] {logging_mixin.py:188} INFO - Produced: weedle with key=bug
[2025-06-06T03:17:13.384+0000] {logging_mixin.py:188} INFO - Produced: kakuna with key=bug
[2025-06-06T03:17:13.847+0000] {logging_mixin.py:188} INFO - Produced: beedrill with key=bug
[2025-06-06T03:17:14.312+0000] {logging_mixin.py:188} INFO - Produced: pidgey with key=normal
[2025-06-06T03:17:14.759+0000] {logging_mixin.py:188} INFO - Produced: pidgeotto with key=normal
[2025-06-06T03:17:15.237+0000] {logging_mixin.py:188} INFO - Produced: pidgeot with key=normal
[2025-06-06T03:17:15.703+0000] {logging_mixin.py:188} INFO - Produced: rattata with key=normal
[2025-06-06T03:17:15.906+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-06T03:17:15.907+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-06T03:17:15.913+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=pokemon_partitioned_to_kafka, task_id=fetch_pokemon_and_send_partitioned_to_kafka, execution_date=20250606T030200, start_date=20250606T031656, end_date=20250606T031715
[2025-06-06T03:17:15.932+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-06T03:17:15.940+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-06T03:17:15.941+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
